Source/JavaScriptCore/inspector/remote/RemoteInspector.h

```diff
@@ -268,6 +271,8 @@ class JS_EXPORT_PRIVATE RemoteInspector final
 #if PLATFORM(COCOA)
     RefPtr<RemoteInspectorXPCConnection> m_relayConnection;
     bool m_shouldReconnectToRelayOnFailure { false };
+
+    bool m_pendingMainThreadInitialization { false };
```

Comment:
hmm should this also be `WTF_GUARDED_BY_LOCK(m_mutex)` (it seems like every use already does)?

Reply:
Sigh. It looks like I got the wires crossed in my brain and made `m_automaticInspectionEnabled` atomic instead of this, which I meant to 🤦🏻

Reply:
aha lol it happens :P

might be worth just making them both `WTF_GUARDED_BY_LOCK(m_mutex)` instead